package.json

```diff
@@ -22,12 +22,13 @@
   },
   "main": "node/index.js",
   "dependencies": {
+    "babel-core": "^5",
     "babel-runtime": "^5",
     "lodash": "^3.3.1",
+    "through": "^2.3.6",
     "tiny-sprintf": "^0.3.0"
   },
   "devDependencies": {
-    "babel-core": "^5",
```

Comment:
Let me know if you want to give what I suggested a try! If not, I can also take care of it.